src/main/java/com/puppycrawl/tools/checkstyle/gui/ParseTreeTablePModel.java

```diff
@@ -242,7 +242,7 @@ public int getIndexOfChild(Object parent, Object child) {
      * @return true if editable
      */
     public boolean isCellEditable(int column) {
-        return getColumnClass(column).equals(ParseTreeTablePModel.class);
+        return getColumnClass(column).equals(ParseTreeTableModel.class);
```

Comment:
@romani Should I return false here and adjust the test accordingly? Or should getColumnClass for column 0 return ParseTreeTablePModel instead of ParseTreeTableModel. That change breaks the UI as I have tested it.